main_window/mapping/mapbox_utils.py

```diff
@@ -262,8 +270,10 @@ def getRow(row):
                     i.getImage(overwrite=overwrite)
                     a = a - 1
 
-        with concurrent.futures.ThreadPoolExecutor() as executor:
-            executor.map(getRow, self.ta)
+        for row in self.ta:
+            getRow(row)
+        # with concurrent.futures.ThreadPoolExecutor() as executor:
+        #     executor.map(getRow, self.ta)
```

Comment:
This needs to be undone i guess

Reply:
My higher-order functions will not be taken away.